include/swift/Parse/Parser.h

```diff
@@ -870,7 +870,7 @@ class Parser {
                            IsFollowingGuard);
   }
   ParserResult<BraceStmt> parseBraceItemList(Diag<> ID);
-  
+
```

Comment:
Unrelated change?

Reply:
It seems like there was trailing white space and xCode automatically trimmed it.